<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiWebScrapper</class>
 <widget class="QWidget" name="UiWebScrapper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Web Scraper Tool</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_WebScraper_RecordsCount">
     <property name="text">
      <string>Records count: 0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QLabel" name="label_49">
     <property name="text">
      <string>Hint: data is automatically parsed when pasted</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="AdvancedTableWidget" name="tableWidget_WebScraper">
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Protocol</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Domain</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>URL</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP Address</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Organization</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>HTTP Response</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_ActiveWorkers">
     <property name="text">
      <string>Active workers: 0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_WebScraper_StartDownload">
     <property name="text">
      <string>Download pages</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_WebScraper_StopDownload">
     <property name="text">
      <string>Stop downloading</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton_WebScraping_StretchCols">
     <property name="text">
      <string>Stretch cols</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButton_WebScraping_Clear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdvancedTableWidget</class>
   <extends>QTableWidget</extends>
   <header>base/extensions/AdvancedTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
